layouts/partials/head.html

```diff
@@ -0,0 +1,53 @@
+{{ $domain_web := .Site.Params.domain_web}}
+
+<head>
+  <meta charset="UTF-8" />
+  <meta
+    name="viewport"
+    content="width=device-width, initial-scale=1, shrink-to-fit=no"
+  />
+
+  <meta property="og:type" content="website" />
+  <meta property="og:description" content="{{ .Site.Params.description }}" />
+  <meta property="og:url" content="{{ $domain_web }}" />
+  <meta property="og:image" content="{{ $domain_web }}/_srcf/images/logo.svg" />
+
+  {{ $title := print .Title " | " .Site.Title}} 
+  {{ if .IsHome }}{{ $title = .Site.Title }}{{ end }}
+
+  <meta property="og:title" content="{{ $title }}" />
+
+  {{ if .IsHome }}
+  <meta name="description" content="{{ .Site.Params.description }}" />
+  <meta property="og:description" content="{{ .Site.Params.description }}" />
+  {{ else }}
+  <meta
+    name="description"
+    content="{{ .Params.summary | default .Description}}"
+  />
+  <meta
+    property="og:description"
+    content="{{ .Params.summary | default .Description}}"
+  />
+  {{ end}}
+
+  <title>{{ $title }}</title>
+
+  <!-- <link href="{{$domain_web}}/_srcf/vendor/css/bootstrap.min.css" rel="stylesheet"> -->
+  <!-- <link href="{{$domain_web}}/_srcf/vendor/css/font-awesome.min.css" rel="stylesheet"> -->
+  <link
+    href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css"
+    rel="stylesheet"
+    integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6"
+    crossorigin="anonymous"
+  />
+  <!-- <link href="{{$domain_web}}/_srcf/srcf-bs.css" rel="stylesheet"> -->
+  <link href="/vendor/font-awesome/css/all.css" rel="stylesheet" />
+  <link href="/css/style.css" rel="stylesheet" />
```

Comment:
This assumes the site's hosted at the domain root, so styles won't load with the site in a subdirectory -- in the absence of `relref`, we can use the base URL:

```suggestion
  <link href="{{ .Site.BaseURL }}/vendor/font-awesome/css/all.css" rel="stylesheet" />
  <link href="{{ .Site.BaseURL }}/css/style.css" rel="stylesheet" />
```

Reply:
Done!